libs/designsystem/kirby-ionic-module/src/kirby-ionic.module.ts

```diff
@@ -1,15 +1,25 @@
 import { NgModule } from '@angular/core';
-import { IonicModule } from '@ionic/angular';
+import { AnimationController, IonicModule, isPlatform } from '@ionic/angular';
 
+const getIonicConfig = () => {
+  let config: any = {
```

Comment:
Consider strong typing - the type can be imported from:
`import { IonicConfig } from '@ionic/core';`
```suggestion
  let config: IonicConfig = {
```

Reply:
Done in f07534032